src/components/pages/Collect/Collect.styles.js

```diff
@@ -0,0 +1,15 @@
+import { RECORD_STATUS_LABELS } from './collectConstants'
+import { Tr } from '../../generic/Table/table'
+import styled from 'styled-components'
+import theme from '../../../theme'
+
+const RECORD_STATUS_COLORS = {
+  [RECORD_STATUS_LABELS.error]: theme.color.cautionColor,
+  [RECORD_STATUS_LABELS.ok]: '#298217',
+  [RECORD_STATUS_LABELS.stale]: theme.color.primaryColor,
+  [RECORD_STATUS_LABELS.warning]: theme.color.warningColor,
+}
```

Comment:
I think I like this approach but it took me a minute to understand it.

Also, why is colour a hex colour, whereas the others are from the theme?

Reply:
Al was specific that he wanted that colour. Im not sure yet how it ties into the theme since its used nowhere else (I search)

I think its good to put certain values into constants - like map layer names for example where you are using them a lot to get and set. It takes away most of the typo bugs, and then you get autocomplete. Also in this case, if the api changes the ‘ok’ status to something else, I can just change it in one place (theoretically). 

Can you think of a way that is easier to understand that still uses constants? Are the value of constants not worth it in you mind?